modules/server/src/test/scala/scaladex/server/util/PageReadProvider.scala

```diff
@@ -0,0 +1,14 @@
+package scaladex.server.util
+
+import play.api.libs.json.Json
+import play.api.libs.json.Reads
+import scaladex.core.model.search.Page
+import scaladex.core.model.search.Pagination
+
+object PageReadProvider {
```

Comment:
This can be reused in future tests that may require a paginated response.

All the test would need to do is bring this `object` into scope, and then declare an implicit instance for whatever
response the `Page` wraps. In this. case, a [JSON reader for `ArtifactResponse`](https://github.com/scalacenter/scaladex/pull/992/files#diff-9523193ad631d9906f11f9ec3dab8b6a641973808d238dd90fde442adc6babb9R27)